scripts/ci/openapi-drift.sh

```
@@ -0,0 +1,128 @@
+if [ "$status" -ne 0 ] && [ "$status" -ne "$EXIT_DRIFT" ]; then
+  cat "$workdir/stderr" >&2
+  fail "oasdiff exited $status — it could not compare the documents (this is not a drift report)"
+fi
+
+if [ "$status" -eq 0 ]; then
+  printf '✅ No breaking changes between the baseline and %s\n' "$upstream_ref"
+  exit 0
+fi
+
+{
+  printf '## Upstream OpenAPI drift: `%s` → `%s`\n\n' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)